filebeat/module/auditd/log/ingest/pipeline.yml

```diff
@@ -176,6 +176,94 @@ processors:
 - set:
     field: event.kind
     value: event
+- set:
```

Comment:
How about going with a painless script & parameters kind of like we do in cloudtrail?

```yaml
  - script:
      lang: painless
      ignore_failure: true
      params:
        CONFIG_CHANGE:
          category:
            - configuration
          type:
            - change
        DAEMON_CONFIG:
          category:
            - configuration
          type:
            - change
      source: >-
        if (ctx?.auditd?.record_type == null) {
            return;
        }
        def hm = new HashMap(params.get(ctx.auditd.record_type));
        hm.forEach((k, v) -> ctx.event[k] = v);
```

Reply:
I like that, will do